package.json

```diff
@@ -11,45 +11,45 @@
     "url": "https://github.com/botpress/botpress/issues"
   },
   "dependencies": {
-    "axios": "^0.15.2",
+    "axios": "^0.18.0",
     "babel-polyfill": "^6.23.0",
     "bluebird": "^3.4.6",
     "body-parser": "^1.15.2",
-    "chalk": "^1.1.3",
+    "chalk": "^2.3.1",
     "commander": "^2.9.0",
     "compression": "^1.7.1",
-    "dotenv": "^4.0.0",
-    "eventemitter2": "^2.1.3",
+    "dotenv": "^5.0.0",
+    "eventemitter2": "^5.0.1",
     "express": "^4.14.0",
     "glob": "^7.1.2",
     "howler": "^2.0.3",
     "js-yaml": "^3.8.4",
-    "jsonwebtoken": "^7.1.9",
-    "knex": "^0.12.6",
+    "jsonwebtoken": "^8.1.1",
+    "knex": "^0.14.4",
     "lodash": "^4.16.4",
     "mkdirp": "^0.5.1",
-    "moment": "^2.15.1",
+    "moment": "^2.20.1",
     "monitorctrlc": "^2.0.1",
-    "ms": "^0.7.1",
+    "ms": "^2.1.1",
     "multer": "^1.3.0",
     "mustache": "^2.3.0",
     "mware": "0.0.3",
     "nanoid": "^1.0.1",
     "node-machine-id": "^1.1.3",
     "nodemon": "^1.12.1",
-    "pg": "^6.1.2",
+    "pg": "^7.4.1",
     "prepend-file": "^1.3.1",
     "prompt": "^1.0.0",
     "prompt-confirm": "^1.2.0",
     "query-string": "^5.0.1",
     "randomstring": "^1.1.5",
     "react-codemirror": "^1.0.0",
-    "react-jsonschema-form": "^1.0.0",
+    "react-jsonschema-form": "^1.0.1",
     "react-toastify": "^3.3.3",
-    "socket.io": "^1.5.0",
-    "socket.io-client": "^2.0.3",
+    "socket.io": "^2.0.4",
```

Comment:
@epaminond Great! I'm assuming these don't have breaking changes and you tested this?

Reply:
@slvnperron , I've made some exploratory testing and I skipped packages that had breaking changes (like `nodemon`). Hope I didn't miss anything.